### DISCUSSION TITLE
VSD centered?

### DISCUSSION THREAD

**Discussion by sofia-bg** [UNC-Project-Open-AAC/OS-DPI#232] [Help]
Working on a layout with a single VSD as an example for the makerspace day at ATIA and I am wondering if there is a way for the images to always auto center? 

I used this code in customize and was able to get it to center: 

* {
  display: block;
  margin-left: auto;
  margin-right: auto;
  width: 95%;
}

But was hoping not to have to teach folks how to customize... 

[makerspace_VSD_ex_v1.zip](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/13662994/makerspace_VSD_ex_v1.zip)

**Comment by sofia-bg** [comment]
Realizing that code isn't a great solution because then it keeps the coordinates showing for some reason 
![Screenshot 2023-12-13 at 11 09 53 AM](https://github.com/UNC-Project-Open-AAC/OS-DPI/assets/92817124/bf37808b-f099-4962-894f-6b37da030c54)

**Comment by gbishop** [comment]
The VSD should be centered (I think). The positioning of the image has been
buggy. It is a tricky problem because I don't know the size of the image
until it is loaded.

Those coordinates should only show when the designer is open. That is a bug.

**Comment by gbishop** [comment]
I've got it fixed. Your Update button should light up and after you click
it you shouldn't need that customize block anymore.

**Reply by sofia-bg** [reply]
awesome thanks

**Reply by sofia-bg** [reply]
I just shifted my hot spots and noticed that now the y axis (and thus 0 for x) has shifted towards the right. It works to use negative numbers, but the cursor doesn't show negative x numbers, just 0 for a large portion of the left side of the screen. Wonder if this will trip other people up?

**Reply by sofia-bg** [reply]
oh gosh i have created more problems with this request, just seeing that now when I shifted my hotspots on the programmer interface they still didn't shift right to the user interface. 
<img width="1128" alt="Screenshot 2023-12-13 at 4 01 22 PM" src="https://github.com/UNC-Project-Open-AAC/OS-DPI/assets/92817124/5f897894-0b05-4695-bced-d17bc3b95043">
<img width="1502" alt="Screenshot 2023-12-13 at 4 01 29 PM" src="https://github.com/UNC-Project-Open-AAC/OS-DPI/assets/92817124/b3a8bf28-e56b-47c7-9fbf-53432b151da5">

[Uploading makerspace_VSD_ex_v2.zip…]()

**Comment by gbishop** [comment]
The hotspots are perfectly aligned for me. Did you remove that customized
block?

**Reply by sofia-bg** [reply]
yes i removed the customization

**Reply by sofia-bg** [reply]
not sure why dropping that link in didn't work - this is the current file i am using: 

[makerspace_VSD_ex_v2.zip](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/13665728/makerspace_VSD_ex_v2.zip)

**Comment by gbishop** [comment]
The hotspots are positioned exactly the same for me in both views.

Refresh your browser.

**Comment by gbishop** [comment]
I can reproduce the issue you're having on my chromebook. Maybe it has to
do with aspect ratio? I'm looking into it.

**Comment by gbishop** [comment]
I have pushed a new version to production that "should" fix it. Let me know.

**Comment by gbishop** [comment]
I made another change this morning that cleans up a problem with the vsd
demo.

**Reply by sofia-bg** [reply]
thanks it is working perfectly for me now!
